src/test/resources/org/prebid/server/it/openrtb2/pubmatic/test-auction-pubmatic-request.json

```diff
@@ -49,9 +49,7 @@
   ],
   "tmax": 5000,
   "regs": {
-    "ext": {
-      "gdpr": 0
-    }
```

Comment:
This change and the one in `test-pubmatic-bid-request.json` i've peeked in many other commits that declared other bidders' support of openRTB 2.6